features/car/restrictions.feature

```diff
@@ -506,3 +506,486 @@ Feature: Car - Turn restrictions
+                |   |
+                |   |
+                |   |
+                |   |
+            a - b - c - - - - - - - - - - - - - - - - - - - f
+                |   | \                                     /
+                i - d - e - - - - - - - - - - - - - - - - -
+            """
+
+        And the ways
+            | nodes | oneway |
+            | ab    | no     |
+            | bc    | no     |
+            | cd    | yes    |
+            | ce    | yes    |
+            | cf    | yes    |
+            | cg    | yes    |
+            | bh    | no     |
+            | fedib | yes    |
+
+       And the relations
+            | type        | way:from | way:via | way:to | restriction   |
+            | restriction | ab       | bc      | ce     | no_right_turn |
+            | restriction | ab       | bc      | cd     | no_right_turn |
+
+       When I route I should get
+            | from | to | route                |
+            | a    | i  | ab,bc,cf,fedib,fedib |
+
+
+    @restriction-way @overlap
+    Scenario: Car - prohibit turn
+        Given the node map
+            """
+            a   j
+            |   |
+            b---i
+            |   |
+            c---h
+            |   |
+            d---g
+            |   |
+            e   f
+            """
+
+        And the ways
+            | nodes | name   | oneway |
+            | ab    | left   | yes    |
+            | bc    | left   | yes    |
+            | cd    | left   | yes    |
+            | de    | left   | yes    |
+            | fg    | right  | yes    |
+            | gh    | right  | yes    |
+            | hi    | right  | yes    |
+            | ij    | right  | yes    |
+            | dg    | first  | no     |
+            | ch    | second | no     |
+            | bi    | third  | no     |
+
+        And the relations
+            | type        | way:from | way:via | way:to | restriction   |
+            | restriction | ab       | bi      | ij     | no_u_turn     |
+            | restriction | bc       | ch      | hi     | no_u_turn     |
+            | restriction | fg       | dg      | de     | no_u_turn     |
+            | restriction | gh       | ch      | cd     | no_u_turn     |
+
+        When I route I should get
+            | from | to | route                  |
+            | a    | j  | left,first,right,right |
+            | f    | e  | right,third,left,left  |
+
+    @restriction
+    Scenario: Car - allow only turn
+        Given the node map
+            """
+            c
+            |
+            |   f
+            |   |
+            b---e
+            |   |
+            a   d
+            """
+
+        And the ways
+            | nodes |
+            | ab    |
+            | bc    |
+            | be    |
+            | de    |
+            | ef    |
+
+        And the relations
+            | type        | way:from | way:via | way:to | restriction  |
+            | restriction | ab       | be      | ef     | only_left_on |
```

Comment:
I am not sure whether I follow, but yes. We evaluate `only_*` without checking the `postfix`. Just as we did for via-nodes.